Makefile

```diff
@@ -1,6 +1,7 @@
 PREFIX=/go
 CMD=crc-driver-libvirt
 DESCRIBE=$(shell git describe --tags)
+CONTAINER_RUNTIME ?= podman
```

Comment:
This is the exactly I had in mind , but I think default should be `docker` here. :+1:

Reply:
Or add another commit to change the circle CI config and have this env set to `docker`

Reply:
> Or add another commit to change the circle CI config and have this env set to docker
Testing this, but I think I'll have to rename back the images from Containerfile to Dockerfile